{{ cookiecutter.repo_name }}/{{ cookiecutter.module_name }}/models/predict.py

```diff
@@ -0,0 +1,26 @@
+from pathlib import Path
+
+from loguru import logger
+from tqdm import tqdm
+import typer
```

Comment:
Do I recall correctly that typer is nicer to use if `rich` is also in the env (but it is an optional depdency)? Should we include that as well?

Reply:
As of 0.12.1 (released last week), typer now includes the optional dependencies by default. You'd instead install `typer-slim` to get the minimal version. 

https://typer.tiangolo.com/release-notes/#0121